src/utils.rs

```diff
@@ -84,19 +76,6 @@ pub async fn get_chain(eth_client: Provider<Http>) -> Result<Chain, Error> {
     })
 }
 
-pub async fn get_delegates_keys_by_orchestrator(
```

Comment:
Was nothing calling this then?

Reply:
I think it was leftover from an old command